src/components/GoogleSearch/GoogleSearch.jsx

```diff
@@ -0,0 +1,20 @@
+export const GoogleSearch = () => {
+  return (
+    <div className="search-section">
+      <form
+        method="get"
+        action="https://www.google.com/search"
+        className="search-form"
+      >
+        <i className="fa-solid fa-magnifying-glass search-icon"></i>
+        <input
+          type="text"
+          name="q"
```

Comment:
you could give some more meaningful name to this input field

Reply:
Google uses the "q" querystring variable as it's search-query to form the URL, so I have to give this name only.